src/client/app/containers/BarChartContainer.ts

```diff
@@ -23,18 +21,26 @@ function mapStateToProps(state: State) {
 			const readingsData = byMeterID[timeInterval.toString()][barDuration.toISOString()];
 			if (readingsData !== undefined && readingsData.readings !== undefined && !readingsData.isFetching) {
 				const label = state.meters.byMeterID[meterID].name;
-				const color = getGraphColor(label);
+				const xData: string[] = [];
```

Comment:
A number of comments were added to LineChartContainer.ts and it might be nice to have the mirror ones here.

Reply:
Addressed in commit 62e075f